packages/scripts/config/webpack.config.js

```diff
@@ -115,6 +112,7 @@ const config = {
 		alias: {
 			'lodash-es': 'lodash',
 		},
+		extensions: [ '.ts', '.tsx', '...' ],
```

Comment:
I found the trick with `...` in the webpck docs. It's a way to include all default extensions.

Reply:
Nice!